server/webserver.go

```diff
@@ -1220,6 +1226,16 @@ func (s *LivepeerServer) StartWebserver(bindAddr string) {
 		}
 	})
 
+	mux.HandleFunc("/latestBlock", func(w http.ResponseWriter, r *http.Request) {
+		b, err := s.LivepeerNode.Eth.LatestBlockNum()
```

Comment:
Generally, I think we might want to start using `Database.LastSeenBlock` instead of `Eth.LatestBlockNum` in order to minimize the number of ad-hoc geth RPC calls. But this is OK for now, later we can fix up all the instances in one fell swoop.